INSTALL.txt

```diff
@@ -1,365 +0,0 @@
-
```

Comment:
Installation documentation is often read at the CLI with limited width, hence text files in a limited width are better readable then fancy formatted Markdownfiles with long lines that turn into a readability nightmare.

I would prefer to keep the INSTALL.txt file for that reason.

Reply:
There's nothing fancy about the Markdown syntax that's as readable as the previous syntax, while adding nice rendering on web interfaces/IDEs. If the long lines are bugging you then I can restore the arbitrary wrapping limit.